spec/features/users/facebook_login_spec.rb

```diff
@@ -1,75 +1,53 @@
 require "rails_helper"
 
 describe "User logs in through facebook" do
-    before(:each) do
-        OmniAuth.config.mock_auth[:facebook] = nil
-
-        OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new(
-            {"provider"=>"facebook",
-            "uid"=>"100626065081257",
-            "info"=>
-            {"email"=>"sharon_vzvvxsf_moidustein@tfbnw.net",
-            "first_name"=>"Sharon",
-            "last_name"=>"Moidustein",
-            "image"=>"http://graph.facebook.com/v3.0/100626065081257/picture"},
-            "credentials"=>
-            {"token"=>
-                "EAAqHzoceqSsBAN90pOElp2umzOEvZBNkOqtMzTL13oZBQMLpaZAH1QfXYanI9hLEmOsYvDhJOpYzEZCObeSagX8DOfiHx24yysYPfuiCjNccgCzZBnZApziZC7kZCPDUTVpqsfmNMWACdlLJeUZCmBXJ3y2j54LrZBZBz5dGliPYmd6RtGo6bv52P7GZBtmY3VmTCrAZD",
-            "expires_at"=>1600736355,
-            "expires"=>true},
-            "extra"=>
-            {"raw_info"=>
-                {"id"=>"100626065081257",
-                "first_name"=>"Sharon",
-                "last_name"=>"Moidustein",
-                "email"=>"sharon_vzvvxsf_moidustein@tfbnw.net"}
-            }})
-        end
-
-  xit "logs in through registration page" do
-        OmniAuth.config.test_mode = true
-
-        first_name = "Sharon"
-        last_name = "Moidustein"
 
+  it "logs in through registration page" do
         visit "/"
         click_on "Sign Up"
         expect(current_path).to eq("/users/new")
-        find('#facebook_sign_in').click
```

Comment:
I think the reason the tests were failing is that it wasn't clicking the actual link, just the CSS element.

Reply:
Now it's looking like these aren't working in the Travis CI build, so I will try to fix that now.

Reply:
Looks like it was because the CSS changed in the most recent master and I forgot to run the tests before pushing.

Reply:
oh gosh, I'm sorry. Testing the css with RSpec has been such a pain in so many projects!

Reply:
Nice catch, thanks for going in an changing all of these!